lib/metasploit/framework/login_scanner/redis.rb

```diff
@@ -24,6 +28,7 @@ class Redis
         WRONG_PASSWORD_SET    = /^-WRONGPASS/i
         INVALID_PASSWORD_SET  = /^-ERR invalid password/i
         OK                    = /^\+OK/
+        DEFAULT_PASSWORD = 'foobared'
```

Comment:
is this an actual default password for redis?

Reply:
This is the default password in the `redis.conf` example from their github.

Reply:
from here? https://github.com/redis/redis/blob/a9267137ee5cbd0908d3844e9d57284e93d85b72/redis.conf#L1045
ok, but it's just an example not an actual default password right? I guess I don't understand why we need a default password for anonymous logins? shouldn't there be no password? 😅

Reply:
I'll remove it, because you're right, we don't need a password/username value to detect the anonymous logic.